src/store/item-store.js

```diff
@@ -55,4 +55,14 @@ store.registerHandler('STUDENT_FOUND', data => {
     }
     store.emitChange();
 });
+
+store.registerHandler('ITEM_DUEDATE_UPDATED', item => {
+    for(let i = 0; i < items.length; i++){
```

Comment:
A little bit of a problem with `.find` is that I need to know the index where the item is stored so I can "save" it. `findIndex` works in this instance. Adjusted.